app/controllers/buffered_articles_controller.rb

```diff
@@ -1,20 +1,20 @@
 class BufferedArticlesController < ApplicationController
   # No authorization required for entirely public controller
+
   def index
-    @article_urls = buffered_article_urls
-    render json: {
-      urls: @article_urls
-    }.to_json
+    render json: { urls: buffered_articles_urls }.to_json
   end
 
-  def buffered_article_urls
-    if Rails.env.production?
-      Article.
-        where("last_buffered > ? OR published_at > ?", 24.hours.ago, 20.minutes.ago).
-        select(:path).
-        map { |a| "https://#{ApplicationConfig['APP_DOMAIN']}#{a.path}" }
-    else
-      Article.all.map { |a| "https://#{ApplicationConfig['APP_DOMAIN']}#{a.path}" }
-    end
+  private
+
+  def buffered_articles_urls
+    relation = if Rails.env.production?
+                 Article.where("last_buffered > ? OR published_at > ?", 24.hours.ago, 20.minutes.ago)
```

Comment:
I'm fine leaving at as is, but generally when AR supports an operation directly, I tend to use it:

```ruby
Article.where("last_buffered > ?", 24.hours.ago).or(Article.where("published_at > ?", 20.minutes.ago))
```

The generated SQL is almost identical, the AR version just adds explicit parenthesis around the two conditions.

Reply:
Definitely +1 on `.or`, the code base predates the addition of `.or()` and I didn't notice while reviewing it :) I'm going to change it